core/modules/layout/css/layout.flexible.admin.css

```
@@ -9,28 +9,33 @@
   line-height: 1.2;
   text-transform: uppercase;
   font-weight: normal;
-  color: #000;
+  color: #000000;
 }
 [dir="rtl"] .layout-flexible-region-top .label {
   padding: 4px 0 4px 12px;
 }
 
+.flexible-row .layout-editor-region {
+  border: 1px solid #cccccc;
+}
+
 .flexible-row .layout-editor-region-title .label {
   font-size: 16px;
 }
 
 .layout--flexible .layout-editor-block {
   box-shadow: none;
+  cursor: row-resize;
 }
 .layout--flexible .layout-editor-block-title {
   padding: 0.4em 1em;
-  margin-bottom: 10px;
+  margin: 0 -.9375rem .9375rem -.9375rem;
```

Sorry @stpaultim ...another coding standard issue with leading zeros for decimal values:
```suggestion
  margin: 0 -0.9375rem 0.9375rem -0.9375rem;
```

I wasn't able to find a specific mention of this in https://docs.backdropcms.org/css-standards or https://www.drupal.org/docs/develop/standards/css/css-formatting-guidelines, however, we don't seem to be doing it like that anywhere else in core.

Omitting the zeros especially makes the negative values seem really odd.